game-core/src/main/java/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -410,6 +415,26 @@ public boolean useTerritoryEffectMarkers() {
     return Boolean.valueOf(mapProperties.getProperty(PROPERTY_MAP_USETERRITORYEFFECTMARKERS, "false"));
   }
 
+  public float getSmallMapTerritoryAlpha() {
+    return Float.valueOf(mapProperties.getProperty(PROPERTY_MAP_SMALLMAPTERRITORYTALPHA, "1.0f"));
+  }
+
+  public int getSmallMapUnitSize() {
+    return Integer.valueOf(mapProperties.getProperty(PROPERTY_MAP_SMALLMAPUNITSIZE, "4"));
+  }
+
+  public Color getSmallMapViewerBorderColor() {
+    return getColorProperty(PROPERTY_MAP_SMALLMAPVIEWERBORDERCOLOR, Color.LIGHT_GRAY);
+  }
+
+  public Color getSmallMapViewerFillColor() {
+    return getColorProperty(PROPERTY_MAP_SMALLMAPVIEWERFILLCOLOR, Color.LIGHT_GRAY);
+  }
+
+  public float getSmallMapViewerFillAlpha() {
+    return Float.valueOf(mapProperties.getProperty(PROPERTY_MAP_SMALLMAPVIEWERFILLALPHA, "0.0f"));
```

Comment:
For all of the object classes for primitives you should probably prefer Float#parseFloat or Integer#parseInt because those methods return the primitive instead of the wrapper object.

Reply:
Good catch was just copying the existing usage but updated all instances in this class.